lib/onebox/mixins/github_auth_header.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Onebox
+  module Mixins
+    module GithubAuthHeader
+      def github_auth_header
+        if SiteSetting.github_onebox_access_token.present?
```

Comment:
Could be seen as a nitpick, but we could use a guard clause here, I think :slightly_smiling_face: (unless the line is too long for syntax tree)

Reply:
Sure makes sense will change :+1: